recurring_contract/__openerp__.py

```diff
@@ -30,7 +30,7 @@
 {
     'name': 'Recurring contract',
     'summary': 'Contract for recurring invoicing',
-    'version': '1.1',
+    'version': '1.4',

```

Comment:
From 1.1 to 1.4 ? Either we increment normally (i.e. 1.2) or we follow the branch version of the whole project (i.e. 1.5).